app/models/concerns/shared/citations.rb

```diff
@@ -36,14 +36,15 @@ module Shared::Citations
     # }
 
     scope :order_by_youngest_source_first, -> {
-      join_str = ActiveRecord::Base.send(:sanitize_sql_array,
-                                         ["LEFT OUTER JOIN citations ON #{related_table_name}.id = " \
-                                          'citations.citation_object_id ' \
-                                          'AND citations.citation_object_type = ? LEFT OUTER JOIN sources ' \
-                                          'ON citations.source_id = sources.id',
-                                          related_class])
-    joins(join_str).group("#{related_table_name}.id")
-      .order(Arel.sql("MAX(COALESCE(sources.cached_nomenclature_date, Date('1-1-0001'))) DESC"))
+      join_str = ActiveRecord::Base.send(
+        :sanitize_sql_array,
+        ["LEFT OUTER JOIN citations ON #{related_table_name}.id = " \
+         'citations.citation_object_id ' \
+         'AND citations.citation_object_type = ? LEFT OUTER JOIN sources ' \
+         'ON citations.source_id = sources.id',
+         related_class])
+      joins(join_str).group("#{related_table_name}.id")
+        .order(Arel.sql("MAX(COALESCE(sources.cached_nomenclature_date, Date('1-1-0001'))) DESC"))
```

Comment:
Looks it works, but perhaps should be '0001-01-01'.
```
psql (12.6 (Ubuntu 12.6-1.pgdg20.04+1))
Type "help" for help.

taxonworks_development=# SELECT DATE('1-1-0001');
    date    
------------
 0001-01-01
(1 row)

taxonworks_development=# SELECT DATE('0001-1-1');
    date    
------------
 0001-01-01
(1 row)
```

Reply:
@LocoDelAssembly this was cosmetic not new.  Let's merge and then come back and check in development?